### PR TITLE
🔥🐛update yahoonativeads script and example [amp-embed type=yahoonativeads]

### PR DESCRIPTION
**Pull request by dysfunc**
Updating the current `yahoonativeads` ad integration
- Switch to new production `native.js`
- Update props, readme, and example

### CODE REVIEW

**Comment by dysfunc** [comment]
@calebcordry :) please

File: ads/yahoonativeads.js [hunk 1]
```diff
@@ -21,26 +21,30 @@ import {loadScript, validateData} from '../3p/3p';
...
-    entries.forEach(function(entry) {
-      if (global.Readmo) {
-        global.Readmo.onViewChange({
-          rects: entry,
-        });
-      }
-    });
-  });
+  global.apiKey = data.key;
+  global.publisherUrl = data.url;
```
Comment (calebcordry) [inline ads/yahoonativeads.js]: I can't see your JS but one thing we generally recommend is to make sure you URL encode these values before trying to send requests.

Comment (calebcordry) [reply ads/yahoonativeads.js]: To be clear - I mean doing this in your external JS if it's not already there.

Comment (dysfunc) [reply ads/yahoonativeads.js]: yes, of course

Comment (dysfunc) [reply ads/yahoonativeads.js]: glad you're calling that out in PRs 👍

**Pull request merged**